### PR TITLE
enhance: Store SparseFloatVector into parquet as JSON string

### PR DESCRIPTION
**Pull request by cydrain**
Issue: #22837

### CODE REVIEW

File: pkg/util/typeutil/schema.go [hunk 1]
```diff
@@ -1601,6 +1602,46 @@ func CreateSparseFloatRowFromJSON(input map[string]interface{}) ([]byte, error)
...
 }
 
+func CreateSparseFloatRowFromJSON(input []byte) ([]byte, error) {
+	var indices []uint32
+	var values []float32
+
+	var vec sparseFloatVectorJSONRepresentation
+	decoder := json.NewDecoder(bytes.NewReader(input))
+	decoder.DisallowUnknownFields()
+	err := decoder.Decode(&vec)
```
Comment (zhengbuqian) [inline pkg/util/typeutil/schema.go:1612]: we should be able to remove `sparseFloatVectorJSONRepresentation` and avoid the duplicate parsing
```
var vec map[string]interface{}

err := decoder.Decode(&vec)
return CreateSparseFloatRowFromMap(vec)
```

Comment (cydrain) [reply pkg/util/typeutil/schema.go:1604]: good

**Pull request merged**